modules/c++/sys/source/OSWin32.cpp

```diff
@@ -281,6 +281,12 @@ size_t sys::OSWin32::getNumCPUs() const
     return info.dwNumberOfProcessors;
 }
 
+size_t sys::OSWin32::getNumPhysicalCPUs() const
+{
+    throw except::NotImplementedException(
```

Comment:
Put a TODO with the reference to the link you put in the PR so that we can look at that if we ever need to implement.  Here, use `// TODO` since not in Doxygen :o)